src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -784,6 +785,27 @@ export class SearchInterface extends RootComponent implements IComponentBindings
     this.responsiveComponents.setMediumScreenWidth(this.options.responsiveMediumBreakpoint);
     this.responsiveComponents.setSmallScreenWidth(this.options.responsiveSmallBreakpoint);
     this.responsiveComponents.setResponsiveMode(this.options.responsiveMode);
+    this.ensureFacetsMobileMode();
+  }
+
+  private ensureFacetsMobileMode() {
+    if (this.root.querySelector('.CoveoFacetsMobileMode')) {
+      return;
+    }
+
+    require.ensure(
```

Comment:
Inspired from the `handleDebugModeChange` a few lines down

Reply:
So basically this will dynamically add the FacetMobileMode DOM node right at the beginning of the SearchInterface?

Has this been tested within Salesforce with Locker just to be sure? Otherwise I can take your branch build it and test it I think.